generators/docker-compose/index.js

```diff
@@ -223,11 +251,37 @@ module.exports = class extends BaseDockerGenerator {
     };
   }
 
-  get writing() {
+  _loading() {
+    return {
+      loadSharedConfig() {
+        this.loadAppConfig();
+        this.loadClientConfig();
+        this.loadServerConfig();
+        this.loadTranslationConfig();
```

Comment:
A custom config loader should be implemented.
Configuration is quite different at this generator and isn't supposed to be the same config file.

Reply:
Just wanted here to load to base properties, in addition to the docker-compose generator ones.
Does it cause any issue?

Reply:
Not sure, but docker-compose is not guarantee to be executed in the application folder, so .yo-rc.json doesn't exists in that folder.
IMO is misleading.

Reply:
You're right, forgot about that... Going to rework it